pkg/stages/steps_install.go

```diff
@@ -149,6 +169,72 @@ func GetInstallStage(sis values.System, logger types.KairosLogger) ([]schema.Sta
 				l4tScript,
 			},
 		},
+		{
+			Name: "Setup NVIDIA L4T repositories",
```

Comment:
@Itxaka I'm not sure if I'm doing this properly, can you validate? Basically, I'm trying to mimic the steps in https://github.com/kairos-io/kairos/blob/master/images/Dockerfile.nvidia-orin-nx but I couldn't find a way to say e.g.

1. rm -rf /etc/apt/sources.list.d/nvidia-l4t-apt-source.list
2. install base packages
3. set nvidia repos
4. install kernel packages in that order

nevertheless this seems to build, so maybe it's ok?

@jordankrp how strict does the order in that dockerfile need to be? 😅